tests/queries/0_stateless/01852_cast_operator.reference

```diff
@@ -13,7 +13,7 @@ SELECT CAST([1, 1 + 1, 1 + 2], \'Array(UInt32)\') AS c
 2010-10-10
 SELECT CAST(\'2010-10-10\', \'Date\') AS c
 2010-10-10 00:00:00
-SELECT CAST(\'2010-10-10\', \'DateTime\') AS c
+SELECT CAST(\'2010-10-10\', \'DateTime(\\\'UTC\\\')\') AS c
```

Comment:
You fixed only a test, but probably we should fix CAST itself? We need to return a beginning of a Date when converting from date to datetime?